pkg/gce-pd-csi-driver/controller.go

```diff
@@ -324,7 +324,8 @@ func (gceCS *GCEControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 	existingDisk, err := gceCS.CloudProvider.GetDisk(ctx, gceCS.CloudProvider.GetDefaultProject(), volKey, gceAPIVersion)
 	if err != nil {
 		if !gce.IsGCEError(err, "notFound") {
-			return nil, common.LoggedError("CreateVolume, failed to getDisk when validating: ", err)
+			// failed to GetDisk, however the Disk may already be created, the error code should be non-Final
+			return nil, common.LoggedError("CreateVolume, failed to getDisk when validating: ", status.Error(codes.Unavailable, err.Error()))
 		}
 	}
 	if err == nil {
```

Comment:
(I hate that you can't comment on lines outside the PR...)

On 345 there's a !ready check that returns an internal error code --- we should probably turn this into a temporary error?

You know, I think this is going to screw up our slo filtering :-/

Reply:
and on line 423

Reply:
done

Reply:
(offline discussion: our slo filtering will be fine)